### DISCUSSION TITLE
🚀 Have you got a blog or demonstration relating to MLSEC?

### DISCUSSION THREAD

**Discussion by shsingh** [OWASP/www-project-machine-learning-security-top-10#51] [Show and tell]
If you have a blog, video or demo that highlights the risks with machine learning security let's post them here.

e.g.

Link: https://example.com
Title: 
Abstract:

**Comment by robvanderveer** [comment]
https://owasp.org/www-project-ai-security-and-privacy-guide/
https://www.youtube.com/watch?v=ABmWHnFrMqI
https://www.youtube.com/watch?v=SLdn3AwlCAk&
https://www.brighttalk.com/webcast/19697/586526

**Comment by mik0w** [comment]
Link: https://hackstery.com
Title: Blog on threats of AI/ML 
Abstract: Blog & newsletter, mostly on LLM security, but some of the topics in the newsletter are related to ML
